website/source/docs/agent/options.html.markdown

```diff
@@ -552,7 +552,7 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
     * <a name="dns_port"></a><a href="#dns_port">`dns`</a> - The DNS server, -1 to disable. Default 8600.
     * <a name="http_port"></a><a href="#http_port">`http`</a> - The HTTP API, -1 to disable. Default 8500.
     * <a name="https_port"></a><a href="#https_port">`https`</a> - The HTTPS API, -1 to disable. Default -1 (disabled).
-    * <a name="rpc_port"></a><a href="#rpc_port">`rpc`</a> - The RPC endpoint. Default 8400.
+    * <a name="rpc_port"></a><a href="#rpc_port">`rpc`</a> - The server RPC endpoint. Default 8400.
```

Comment:
Hi @romansky thanks for opening a PR. This one should be CLI as well, server RPC is `server_rpc_port` on line 558.